stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -678,6 +676,38 @@ public class MessageListView : ConstraintLayout, IMessageListView {
         this.onStartThreadListener = onStartThreadListener
     }
 
+    public fun interface MessageClickListener {
+        public fun onMessageClick(message: Message)
+    }
+
+    public fun interface MessageRetryListener {
+        public fun onRetryMessage(message: Message)
+    }
+
+    public fun interface MessageLongClickListener {
+        public fun onMessageLongClick(message: Message)
+    }
+
+    public fun interface AttachmentClickListener {
+        public fun onAttachmentClick(message: Message, attachment: Attachment)
+    }
+
+    public fun interface GiphySendListener {
+        public fun onGiphySend(message: Message, action: GiphyAction)
+    }
+
+    public fun interface UserClickListener {
+        public fun onUserClick(user: User)
+    }
+
+    public fun interface ReadStateClickListener {
+        public fun onReadStateClick(reads: List<ChannelUserRead>)
+    }
+
+    public fun interface ReactionViewClickListener {
+        public fun onReactionViewClick(message: Message)
+    }
```

Comment:
Added new interfaces here so that the new `MessageListView` doesn't depend on the old one